examples/02_fit_predict_plot_employee_salaries.py

```diff
@@ -130,11 +129,15 @@ def make_pipeline(encoding_method):
 # Plotting the results
 # --------------------
 # Finally, we plot the scores on a boxplot:
-# We notice that the MinHashEncoder performs poorly compared to other encoding
-# methods. There are two reasons for that: the MinHashEncoder performs better
-# with tree-based models than linear models (see example 03), and the
-# low-dimensionality of encodings (increasing n_components improves
-# performances.
+# We notice that the MinHashEncoder does not performs as well compared to 
+# other encoding methods.
+# There are two reasons for that: the MinHashEncoder performs better
+# with tree-based models than linear models (
+# :ref:`see example 03<sphx_glr_auto_examples_03_fit_predict_plot_midwest_survey.html#sphx-glr-auto-examples-03-fit-predict-plot-midwest-survey-py>`)
```

Comment:
The pointer to the ref is actually not the name of the file. You need to look at the source code of the generated file to figure it out. I believe that it is `sphx_glr_auto_examples_03_fit_predict_plot_midwest_survey.py`

Reply:
I did not find any occurrence of "sphx_glr" in the source code :/ 

It look likes the :ref: in exemple 05 is also broken 
https://github.com/dirty-cat/dirty_cat/blob/47c5b7650c92f292234dd79cbdc30997fe31dc70/examples/05_scaling_non_linear_models.py#L55